modules/cnv-troubleshooting-incorrect-policy-config.adoc

```diff
@@ -0,0 +1,213 @@
+worker-2.ens01-bridge-testfail         FailedToConfigure
+worker-3.ens01-bridge-testfail         FailedToConfigure
+----
+
+. Viewing one of the Enactments we can look at the traceback. To make the output simpler for this example, we'll pipe the output to grep and search for 'error' without case sensitivity:
++
+----
+$ oc get nnce worker-1.ens01-bridge-testfail -o jsonpath='{.status.conditions[?(@.type=="Failing")].message}'
+----
++
+This returns a large traceback that has been edited for brevity:
++
+----
+error reconciling NodeNetworkConfigurationPolicy at desired state apply: , failed to execute nmstatectl set --no-commit --timeout 480: 'exit status 1' '' 
+...
+libnmstate.error.NmstateVerificationError: 
+desired
+=======
+---
+name: br1
+type: linux-bridge
+state: up
+bridge:
+  options:
+    group-forward-mask: 0
+    mac-ageing-time: 300
+    multicast-snooping: true
+    stp:
+      enabled: false
+      forward-delay: 15
+      hello-time: 2
+      max-age: 20
+      priority: 32768
+  port:
+  - name: ens01
+description: Linux bridge with the wrong port
+ipv4:
+  address: []
+  auto-dns: true
+  auto-gateway: true
+  auto-routes: true
+  dhcp: true
+  enabled: true
+ipv6:
+  enabled: false
+mac-address: 01-23-45-67-89-AB
+mtu: 1500
+
+current
+=======
+---
+name: br1
+type: linux-bridge
+state: up
+bridge:
+  options:
+    group-forward-mask: 0
+    mac-ageing-time: 300
+    multicast-snooping: true
+    stp:
+      enabled: false
+      forward-delay: 15
+      hello-time: 2
+      max-age: 20
+      priority: 32768
+  port: []
+description: Linux bridge with the wrong port
+ipv4:
+  address: []
+  auto-dns: true
+  auto-gateway: true
+  auto-routes: true
+  dhcp: true
+  enabled: true
+ipv6:
+  enabled: false
+mac-address: 01-23-45-67-89-AB
+mtu: 1500
+
+difference
+==========
+--- desired
++++ current
+@@ -13,8 +13,7 @@
+       hello-time: 2
+       max-age: 20
+       priority: 32768
+-  port:
+-  - name: ens01
++  port: []
+ description: Linux bridge with the wrong port
+ ipv4:
+   address: []
+  line 651, in _assert_interfaces_equal\n    current_state.interfaces[ifname],\nlibnmstate.error.NmstateVerificationError:
+----
++
+The `NmstateVerificationError` lists the `desired` Policy configuration, the `current` configuration of how that has been applied to the node, and the `difference` highlighting the parameters that do not match. In this example, the `port` is included in the `difference`, which suggests that the problem is the port configuration in the Policy. 
+
+. To ensure that the Policy is configured properly, we can view the network configuration for one or all of the nodes by requesting the `NodeNetworkState`. The following command returns the network configuration for the `master-1` node:
++
+----
+$ oc get nns master-1 -o yaml
+----
++
+The truncated section of the response reveals the interface name on the nodes is `ens1`, however the failed Policy incorrectly uses `ens01`:
```

Comment:
"The truncated section" is confusing. Is it a truncated section in the output, or are you referring to the following truncated section?

s/response/output (I think)

s/reveals/shows that

s/however/but